build.gradle

```diff
@@ -30,7 +30,7 @@ subprojects {
         buildToolsVersion "26.0.2"
 
         defaultConfig {
-            minSdkVersion 14
+            minSdkVersion 16
```

Comment:
Why u do this? 😢
Goal is to keep this at the same Min SDK as the support library